include/libtorrent/torrent_handle.hpp

```diff
@@ -835,6 +835,7 @@ namespace libtorrent
 		void queue_position_down() const;
 		void queue_position_top() const;
 		void queue_position_bottom() const;
+
 		// set the new position in queue starting by 0. If given position does
```

Comment:
please say: "updates the position in the queue for this torrent. The relative order of all other torrents remain intact but their numerical queue position shifts to make space for this torrent's new position"